test/server/compiler-test.js

```diff
@@ -47,6 +47,39 @@ describe('Compiler', function () {
     });
 
     describe('ES-next', function () {
+        it('Should compile test defined in separate module if option is enabled', function () {
+            const sources = [
```

Comment:
Also add tests for negative cases: 
If there are no `test` and/or `fixture` definitions into imported files, then TestCafe should report clear error messages.

Reply:
ok